test/bigtest/interactors/circulation-rules.js

```diff
@@ -121,6 +121,20 @@ const scrollingOffset = 3;
     }).run();
   });
 
+  changeValue = action(function (value, showHint = true) {
```

Comment:
I'm not sure that we need to create a new method for it. At least, we can try to reuse a setValue because there is a very similar body for now.

@SergiyVSergiyenko what do you think about it?